### DISCUSSION TITLE
Fingerprinting only showing Hydrophobic Interactions

### DISCUSSION THREAD

**Discussion by davidcooper7** [chemosim-lab/ProLIF#95] [General]
Hello. I'm having trouble getting the fingerprints to show me any interactions other than hydrophobic. I know there are hydrogen bond interactions(see image from pyMOL). Any help in the right direction would be great. 
![help1](https://user-images.githubusercontent.com/117563200/200159558-0f116a36-aece-43fc-87a9-2d4effa568de.png)
![help2](https://user-images.githubusercontent.com/117563200/200159559-6ad330b1-def3-4276-a53a-cf6acf1c2d57.png)

**Comment by tmenzonatto** [comment]
Any answers to that?

**Reply by cbouy** [reply]
See issue #96 , if that doesn't help please open a separate issue providing more details about what kind of inputs you're using, which version of ProLIF...etc.
